### ISSUE TITLE
Green flash message box in the new registration page

### ISSUE THREAD

**Issue by wintermeyer** [wintermeyer/vutuv#321]
The flash message leaves a green box on top of the background image. This shouldn't be there.

![bildschirmfoto 2016-11-20 um 11 26 27](https://cloud.githubusercontent.com/assets/69770/20462130/4a124878-af14-11e6-828b-6e183bae5c7b.png)

**Comment by cjfreeze** [comment]
I didn't see this issue before I created a duplicate of it in #328, but I am closing this one because the other has more information and is part of an earlier milestone. I believe this issue should be finished before 1.0.
